src/code/z_lights.c

```diff
@@ -1,10 +1,14 @@
-#include "global.h"
+#include "z64light.h"
+
 #include "sys_cfb.h"
+#include "z64.h"
+#include "functions.h" // for SkinMatrix_Vec3fMtxFMultXYZ and Actor_GetProjectedPos
```

Comment:
Same here (and a few other places)

Reply:
Sorry, those were notes for myself. I was using them to remember why those global headers were included and consider if I could make a header for them